test/blackbox-tests/test-cases/coq/failed-config.t/run.t

```diff
@@ -139,10 +133,8 @@ Should fail.
   File "dune", line 4, characters 8-21:
   4 |   (echo %{coq:coqlib})))
               ^^^^^^^^^^^^^
-  Error: Could not expand %{coq:coqlib} as running coqc --config failed.
+  Error: Could not expand %{coq:coqlib} as running coqc failed.
   $TESTCASE_ROOT/bin/coqc --config failed with exit code 1.
-  Hint: coqc --config requires the coq-stdlib package in order to function
```

Comment:
I would have preferred for this hint to stay, but I see that it's not so easy to reuse the error in both the version and config cases so I guess this is fine for now.

Reply:
Indeed let's not delay the PR on this, but for sure it would be helpful to users to be more specific about errors in this code. This kind of small stuff tends to matter a lot w.r.t. usability.